pkg/session_recording/recordings.jsx

```diff
@@ -571,14 +583,17 @@
 
         handleDateSinceChange(date) {
             this.setState({dateSince: date});
+            cockpit.location.go([], $.extend(cockpit.location.options, { dateSince: date }));
```

Comment:
Wouldn't it be better to do `setState` in `onLocationChanged`, same as recording ID is retrieved?
That way it would work even if the URL was changed some other way, e.g. by hand.